src/e2e/specs/landing.spec.ts

```diff
@@ -158,6 +158,73 @@ test.describe(`${process.env.E2E_TEST_ENV} - Verify the Landing Page content`, (
     await expect(landingPage.privacyLink).toBeVisible();
     await expect(landingPage.githubLink).toBeVisible();
   });
+
+  test("Verify the 'Get data removal' button UI and functionality for both yearly and montly options", async ({
+    landingPage,
+    purchasePage,
+  }) => {
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463525",
+    });
+
+    await expect(landingPage.getDataRemoval).toBeVisible();
+    await expect(landingPage.getDataRemovalMonthly).toBeVisible();
+    await expect(landingPage.getDataRemovalYearly).toBeVisible();
+
+    // Monthly
+    await landingPage.getDataRemovalMonthly.click();
+    await landingPage.getDataRemoval.click();
+    await purchasePage.subscriptionHeader.waitFor();
+    await expect(purchasePage.planDetails).toContainText("monthly");
+    await expect(purchasePage.subscriptionHeader).toBeVisible();
+
+    // Yearly
+    await landingPage.open();
+    await landingPage.getDataRemoval.click();
+    await purchasePage.subscriptionHeader.waitFor();
+    await expect(purchasePage.planDetails).toContainText(
+      `${process.env.E2E_TEST_ENV === "prod" ? "yearly" : "every ⁨2⁩ months"}`,
+    );
+  });
+
+  test('Verify the "Get free scan" corresponding email fields', async ({
+    landingPage,
+    authPage,
+  }) => {
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463504",
+    });
+
+    await landingPage.monitorHeroFormEmailInputField.fill("invalid");
+    await landingPage.monitorHeroFormInputSubmitButton.click();
+    // Stays on same page
+    await expect(landingPage.monitorHeroFormEmailInputField).toBeVisible();
+
+    const randomEmail = `${Date.now()}_tstact@restmail.net`;
```

Comment:
Doubt it matters much for anything, but the other restmail accounts looked like:
```
`_${Date.now()}@restmail.net`;
```